### PR TITLE
Additional intro blockchain doc

### PR DESCRIPTION
**Pull request by GK3077**
**With respect to *Blockchain* File**

### Describe your change:

* [ ] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [x] Documentation change?

### Checklist:
* [x] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [x] This pull request is all my own work -- I have not plagiarized.
* [x] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All new Python files are placed inside an existing directory.
* [ ] All filenames are in all lowercase characters with no spaces or dashes.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
* [ ] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{6642}`.

### CODE REVIEW

File: blockchain/README.md [hunk 1]
```diff
@@ -1,7 +1,44 @@
...
+
+Now lets assume every branch has a copy of the ledger and then once anything is added to the ledger by anyone branch it is gonna automatically reflect in all other ledgers available in other branch. This is done using Peer-to-peer network.
+
+So this means even if tampered in one branch we can find out. If one branch is hacked we can be alerted and get to safeguard other branch. Now, assume these branches as computer or nodes and the ledger is a transaction record or digital receipt. so this is the concept of Digital Ledger Technology.
+
+*Do all node needs all the info in others nodes? Does this increase storage space in all nodes?*
+
+## Blocks
+
+In short a block is nothing but collections of records with a labelled header. These are connected cryptographically. Once a new block is added to a chain previous block is connected or precisely said as locked and will remain unaltered. we will see how this works once we dig deep into working mechanisms.
```
Comment (CaedenPH) [inline blockchain/README.md:29]:
```suggestion
In short a block is nothing but collections of records with a labelled header. These are connected cryptographically. Once a new block is added to a chain previous block is connected or precisely said as locked and will remain unaltered. We will see how this works once we dig deep into working mechanisms.
```

**Review by CaedenPH — CHANGES_REQUESTED** [review]
A lot of the sentences you have written don't make grammatical sense. Please review your sentences and fix any grammatical/punctuational mistakes.

If you need any help don't hesitate to ask

File: blockchain/README.md (hunk 1 above)
Comment (GK3077) [reply blockchain/README.md]: Hi, Thanks for the feedback. As per above suggestions, I did changes to the document and committed it to this PR. Please revert back if any other changes are required in the document.

**Review by CaedenPH — APPROVED** [review]

**Comment by GK3077** [comment]
Thank you for acknowledging my work. Hope to contribute more to the project.

**Review by CaedenPH — APPROVED** [review]
@cclauss Can you review please

File: blockchain/README.md [hunk 2]
```diff
@@ -1,7 +1,44 @@
 # Blockchain
 
-A Blockchain is a type of distributed ledger technology (DLT) that consists of growing list of records, called blocks, that are securely linked together using cryptography.
+A Blockchain is a type of distributed ledger technology **(DLT)** that consists of growing list of records, called **blocks**, that are securely linked together using **cryptography**.
```
Comment (cclauss) [inline blockchain/README.md:3]:
```suggestion
A Blockchain is a type of **distributed ledger** technology (DLT) that consists of growing list of records, called **blocks**, that are securely linked together using **cryptography**.
```

Comment (GK3077) [reply blockchain/README.md]: Thank you for the feedback. I have made the requested changes in this file. Please revert back if any other changes are required in the document.

**Pull request merged**